sys/linux/bpf.txt

```diff
@@ -97,11 +97,30 @@ bpf_prog {
 	prog_name		array[const[0, int8], BPF_OBJ_NAME_LEN]
 	prog_ifindex		ifindex[opt]
 	expected_attach_type	flags[bpf_attach_type, int32]
+	btf_fd			fd_btf
+	func_info_rec_size	len[func_info, int32]
```

Comment:
From the code this looks like size of 1 record rather than array size. Please double check.

Reply:
can we use grammar like "func_info_rec_size sizeof(func_info)"?

Reply:
Yes, this can be done this way:

https://github.com/google/syzkaller/blob/70d776a21acfa3a596bff57fc1a9a4f7998bd4a8/sys/linux/binfmt.txt#L120-L121